docs/guide/using-plugins.md

```diff
@@ -0,0 +1,84 @@
+# Using Plugins
+
+Vite can be extended using plugins, which are based on Rollup's well-designed plugin interface with a few extra vite-specific options. This means that Vite users can rely on the mature ecosystem of rollup plugins, while also being able to extend the dev server and SSR functionality as needed.
+
+## Adding a Plugin
+
+To use a Plugin, it needs to be added to the `devDependencies` of the project and included in the `plugins` array in the `vite.config.js` config file. For example, to provide support for legacy browsers, the official [@vitejs/plugin-legacy](https://github.com/vitejs/vite/tree/main/packages/plugin-legacy) can be used:
+
+```
+$ npm i -D @vitejs/plugin-legacy
+```
+
+```js
+// vite.config.js
+import legacy from '@vitejs/plugin-legacy'
+
+export default {
+  plugins: [
+    legacy({
+      targets: ['defaults', 'not IE 11']
+    })
+  ]
+}
+```
+
+`plugins` also accept presets including several plugins as a single element. This is useful for complex features (like framework integration) that are implemented using several plugins. The array will be flattened internally.
+
+Falsy plugins will be ignored, which can be used to easily activate or deactivate plugins.
+
+## Finding Plugins
+
+:::tip NOTE
+Vite aims to provide out-of-the-box support for common web development patterns. Before searching for a Vite or Compatible Rollup plugin, check out the [Features Guide](../guide/features.md). A lot of the cases where a plugin would be needed in a Rollup project are already covered in Vite.
+:::
+
+Check out the [Plugins section](../plugins) for information about official plugins. Community Plugins are listed in [awesome-vite](https://github.com/vitejs/awesome-vite#plugins). For Compatible Rollup Plugins, check out [Vite Rollup Plugins](https://vite-rollup-plugins.patak.dev) for a list of compatible official rollup plugins with usage instructions or the [Rollup Plugin Compatibility section](../guide/api-plugin#rollup-plugin-compatibility) in case it is not listed there.
+
+You can also find plugins that follow the [recommended conventions](./api-plugin.md#conventions) using a [npm search fro vite-plugin](https://www.npmjs.com/search?q=vite-plugin&ranking=popularity) for Vite Plugins or a [npm search for rollup-plugin](https://www.npmjs.com/search?q=rollup-plugin&ranking=popularity) or a [npm search fro vite-plugin](https://www.npmjs.com/search?q=vite-plugin&ranking=popularity) for Rollup Plugins.
+
+## Enforcing Plugin Ordering
+
+For compatibility with some Rollup Plugins, it may be needed to enforce the order of the plugin or only apply at build time. This should be an implementation detail for Vite Plugins. You can enforce the position of a Plugin with the `enforce` modifier:
+
+- `pre`: invoke plugin before Vite core plugins
+- default: invoke plugin after Vite core plugins
+- `post`: invoke plugin after Vite build plugins
```

Comment:
**suggestion:** Maybe just write
```
- `post`: (default) invoke plugin after Vite build plugins
```

Reply:
The default is `undefined`

```
enforce?: 'pre' | 'post'
```

Reply:
Oh and it's after core and not after build :eyes: